### PR TITLE
Shrunk seeder files, made fixes team leads requested

### PR DESCRIPTION
**Pull request by dstad418**
### Description
Cut down seeder files to around 300 instances of data.
Reset the DB to test these changes, with no errors noted in docker logs or relational breaks
Changed output to only show in **_'prd'_**

### Risks
May cause issues if user doesn't re-migrate, and composes the changes with previous DB images. 
Potential relational breaks, or errors as noted in cloud deployment

### Validation
Brought the DB down and up again, no issues noted.

### Operating System
MacOS, M2 ARM

### CODE REVIEW

File: server/prisma/migrations/20231201205517_updated_pph_tables/migration.sql
Comment (henrykaus) [inline server/prisma/migrations/20231201205517_updated_pph_tables/migration.sql:1]: Do you know why this migration was created?

Comment (dstad418) [reply server/prisma/migrations/20231201205517_updated_pph_tables/migration.sql:1]: Not certain, I pulled from main before I made the changes, but that was a migration that was required for me to build the DB.
Looks like it's to help with integrity for the 'eventtickets' model, and to make the fk adjustments more robust.

Comment (henrykaus) [reply server/prisma/migrations/20231201205517_updated_pph_tables/migration.sql:1]: Interesting, well it lines up with the current postgres schema so I don't see a problem

Comment (benfaustin95) [reply server/prisma/migrations/20231201205517_updated_pph_tables/migration.sql:1]: I don't think that it actually changes anything so it should be good. 👍

**Review by henrykaus — APPROVED** [review]

**Pull request merged**